content/docs/extensions/pg-extensions.md

```diff
@@ -5,84 +5,85 @@ redirectFrom:
+| [intarray](https://www.postgresql.org/docs/15/intarray.html)               | 1.5         | 1.5         |  1.5         |                                                                                                                   |
+| [ip4r](https://github.com/RhodiumToad/ip4r)                     | 2.4.2                    | 2.4.2                    |   2.4.2                    |                                                                                                                   |
+| [isn](https://www.postgresql.org/docs/15/isn.html)                      | 1.2              | 1.2              |   1.2              |                                                                                                                   |
+| [lo](https://www.postgresql.org/docs/15/lo.html)                       | 1.1               | 1.1               |  1.1               |                                                                                                                     |
+| [ltree](https://www.postgresql.org/docs/15/ltree.html)                    | 1.2            | 1.2           |   1.2           |                                                                                                                  |
+| [moddattime (spi)](https://www.postgresql.org/docs/current/contrib-spi.html)         | 1.0 | 1.0  |   1.0  |                                                                                                                  |
+| [neon_utils](/docs/extensions/neon-utils)               | 1.0                              | 1.0                               |     1.0              |                                                                                                                   |
+| [pg_embedding](https://github.com/neondatabase/pg_embedding)             | 0.3.50           | 0.3.5  | 0.3.5            | Install with `CREATE EXTENSION embedding;`                                                                         |
+| [pg_graphql](https://github.com/supabase/pg_graphql)               | 1.1.0                 | 1.1.0                  |  1.1.0                  |                                                                                                                   |
+| [pg_hashids](https://github.com/iCyberon/pg_hashids)                | 1.2.1                | 1.2.1                  |  1.2.1                  |                                                                                                                  |
+| [pg_hint_plan](https://github.com/ossc-db/pg_hint_plan)             | 1.5                  | 1.5                  |  1.5                  |                                                                                                                   |
```

Comment:
We have different versions of `pg_hint_plan` for each Postgres version:
- 1.4.1 for Postgres 14
- 1.5.0 for Postgres 15
- 1.6.0 for Postgres 16

https://github.com/neondatabase/neon/blob/dce91b33a4ce24b1526ef1c39a95761cb0d7da2b/Dockerfile.compute-node#L421-L437

Reply:
Thank you. Updated.